### PR TITLE
Rectify lies about escape.

### PR DESCRIPTION
**Pull request by dereuromark**
For https://github.com/cakephp/cakephp/pull/9358

### CODE REVIEW

**Comment by dereuromark** [comment]
I think I branched off the wrong branch (master = 2.x) :/

File: en/controllers/components/flash.rst [hunk 1]
```diff
@@ -85,10 +85,21 @@ The code above will use the warning.ctp element under **plugins/PluginName/src/T
 for rendering the flash message.
 
 .. note::
+By default, CakePHP escapes the content in flash messages for security reasons. If you
+    are using any request or user data in your flash messages those are escaped and therefore safe to be printed.
+    If you want to output HTML, you need to pass in an `escape` param and also adjust the templates to allow disabling escaping when such a param is passed.
```
Comment (ravage84) [inline en/controllers/components/flash.rst]: Aren't those lines a bit long?

Comment (markstory) [inline en/controllers/components/flash.rst]: And not indented :smile:

Comment (dereuromark) [inline en/controllers/components/flash.rst]: Someone wants to fix it up? That would be awesome.

**Comment by andtxr** [comment]
Fixed.

File: en/controllers/components/flash.rst [hunk 2]
```diff
@@ -85,10 +85,25 @@ The code above will use the warning.ctp element under **plugins/PluginName/src/T
 for rendering the flash message.
 
 .. note::
+    By default, CakePHP escapes the content in flash messages for security
+    reasons. If you are using any request or user data in your flash messages
+    those are escaped and therefore safe to be printed. If you want to output
+    HTML, you need to pass in an `escape` param and also adjust the templates to
```
Comment (dereuromark) [inline en/controllers/components/flash.rst]: Do we need double char here for code pieces? I forgot this is not real markdown :P

**Pull request merged**